framework/web/UrlManager.php

```diff
@@ -143,11 +143,6 @@ class UrlManager extends Component
      */
     public $normalizer = false;
 
-    /**
-     * @var string the cache key for cached rules
-     * @since 2.0.8
-     */
-    protected $cacheKey = __CLASS__;
```

Comment:
@dynasource  first this is protected, second I implemented a getter with same name